util/patricia_trie/src/triedbmut.rs

```diff
@@ -77,36 +79,38 @@ enum Node<H: Hasher> {
 	Branch(Box<[Option<NodeHandle<H>>; 16]>, Option<DBValue>)
 }
 
-impl<H: Hasher> Node<H> {
+impl<I> Node<I> where I: AsRef<[u8]> + AsMut<[u8]> + Default + HeapSizeOf + Debug + PartialEq + Eq + Hash + Send + Sync + Clone + Copy {
```

Comment:
Does `I` stand for "item" or what was the thinking here? Maybe `O` for "Out" would help the reader get the right association?

Reply:
it is just an arbitrary uppercase character, I can change it to `O` if it makes it more readable :)